### PR TITLE
adds business logic to visualization

### PR DESCRIPTION
**Pull request by sambhxv**
Based on the task description of [AB-03291](https://erp.ambibuzz.com/app/task/AB-03291). 
image

### CODE REVIEW

File: ampower_visualize/ampower_visualize/page/product_traceability/product_traceability.js
Comment (amitgandhi12) [inline ampower_visualize/ampower_visualize/page/product_traceability/product_traceability.js:1]: add filters in Doctytpe field don't show doctype which are not relevant

File: ampower_visualize/ampower_visualize/page/product_traceability/product_traceability.js [hunk 1]
```diff
@@ -198,65 +205,135 @@ const refresh_list_properties = () => {
...
 	frappe.call({
-		method: 'ampower_visualize.ampower_visualize.page.product_traceability.product_traceability.get_linked_documents',
-		args: {
-			doctype: doctype,
-			docname: document_name
-		},
+		method: method_type,
+		args: { document_name: document_name },
 		callback: function (r) {
+			console.log(r.message)
```
Comment (amitgandhi12) [inline ampower_visualize/ampower_visualize/page/product_traceability/product_traceability.js]: remove console logs

File: ampower_visualize/ampower_visualize/page/product_traceability/product_traceability.js
Comment (sambhxv) [reply ampower_visualize/ampower_visualize/page/product_traceability/product_traceability.js:1]: done

File: ampower_visualize/ampower_visualize/page/product_traceability/product_traceability.js (hunk 1 above)
Comment (sambhxv) [reply ampower_visualize/ampower_visualize/page/product_traceability/product_traceability.js]: done

**Review by amitgandhi12 — APPROVED** [review]

**Pull request merged**